tables_en/5-02-03.csv

```diff
@@ -0,0 +1,16 @@
+notation,domain,tags,name,description
+,ocean,"ocean,chlorophyll a,CDOM, backscatter","Fluorometry, optical backscattering",Measurement of emitted fluorescent light and particulate backscattering coefficient
+,"atmosphere,ocean","atmosphere,ocean,temperature",Mercury-based thermometry,Measurement of temperature using a mercury based column
+,"atmosphere,ocean","atmosphere,ocean,temperature",Fluid-based thermometry (excluding mercury),Measurement of temperature using a fluid based column (other than mercury)
+,"atmosphere,ocean","atmosphere,ocean,temperature",Electrical resistivity,Measurement derived from the measurement of electrical resistivity
+,ocean,"ocean,salinity",Electrical conductivity,Measurement derived from the measurement of electrical conducivity
+,ocean,"ocean,salinity",Inductivity,Measurement derived from the measurement of magnetic inductance
+,ocean,"ocean,pressure",Piezoresistivity,Measurement of the change in electrical resistance of a material under mechanical stress.
+,ocean,"ocean,waves,direction,period,height",HF echo doppler spectra analysis method (HR radar),Remote sensing of surface currents employing radars frequencies
+,ocean,"ocean,current,direction,speed",GPS fixes sequence comparison,Measurement derived from the difference between two successives GPS positions
+,ocean,"ocean,dissolved oxygen",Clark electrode polarographic method,The dissolved oxygen concentration by counting the number of oxygen molecules per second
+,,,,(flux) that diffuse through the membrane from the ocean environment to the working electrode
+,ocean,"ocean,dissolved oxygen",Oxygen luminescence quenching method,The fluorescent indicator is a special complex embedded in a gas permeable sensing foil that is exposed to the surrounding water. The sensing foil is excited by modulated light; the sensor measures the phase of the returned light.
+,ocean,"ocean,pH",Potentiometric method,ion-sensitive field-effect transistor that responds to proton activity and
+,,,,a Ag/AgCl reference electrode
```

Comment:
@anthoninlize Is "a Ag/AgCl reference electrode" supposed to be a part of the definition for "Potentiometric"? Right now it appears in a new row preceded by empty cells.

Reply:
Ahhh, indeed. I can update. What's the best way to handle this there? Closing this pull request and create another one with the fix?